src/content/docs/apis/nerdgraph/examples/async-queries-nrql-tutorial.mdx

```diff
@@ -0,0 +1,67 @@
+---
+title: "NerdGraph tutorial: Run NRQL queries asynchronously"
+tags:
+  - APIs
+  - NerdGraph
+  - Examples
+metaDescription: Use New Relic NerdGraph to make asynchronous NRQL queries that have longer query duration limits.
+---
+
+You can use our NerdGraph API to run an asynchronous NRQL query. This is useful for running queries that have less risk of being interrupted by browser or HTTP connection timeout limits or other issues. It's also especially applicable for running queries that may take a long time to complete.
+
+## Requirements [#requirements]
+
+The normal [NRQL query duration limit](/docs/query-your-data/nrql-new-relic-query-language/get-started/rate-limits-nrql-queries) is one minute. To run queries with a 10-minute limit, you must have [Data Plus](/docs/accounts/accounts-billing/new-relic-one-pricing-billing/data-ingest-billing#data-plus). (edited) 
```

Comment:
Do you want (edited) to be here? Calling it out just in case

Reply:
ha. whoops